ide/Espressif/ESP-IDF/setup_win.bat

```diff
@@ -0,0 +1,415 @@
+@echo off
+REM Expect the script at /path/to/wolfssh/IDE/Espressif/ESP-IDF/
+
+::******************************************************************************************************
+::******************************************************************************************************
+echo;
+echo wolfSSH (Secure Shell) Windows Setup. Version 0.1c
+echo;
+echo This utility will copy a static snapshot of wolfSSH files to the ESP32-IDF component directory.
+echo;
+echo You must first have installed wolfSSL component for this SSH component to function properly.
+echo;
+echo If you wish to keep your component library fresh with git pull, consider hard link with mklink.
+echo;
+echo    "mklink [[/d] | [/h] | [/j]] <link> <target>"
+echo;
+::******************************************************************************************************
+::******************************************************************************************************
+SET COPYERROR=false
+pause
+
+:: if there's a setup.sh, we are probably starting in the right place.
+if NOT EXIST "setup.sh" (
+  echo Please run this script at /path/to/wolfssh/IDE/Espressif/ESP-IDF/
+  goto :ERR
+)
+
+:: if there's also a default wolfssh_espressif_semaphore.md, we are very likely starting in the right place.
+if NOT EXIST "wolfssh_espressif_semaphore.md" (
+  echo Please run this script at /path/to/wolfssh/IDE/Espressif/ESP-IDF/
+  goto :ERR
+)
+
+:: see if there was a parameter passed for a specific EDP-IDF directory
+:: this may be different than the standard ESP-IDF environment (e.g. VisualGDB)
+if "%1" == "" (
+    if "%IDF_PATH%" == "" (
+        echo;
+        echo ERROR: Specify your ESP-IDF path as a parameter or run from ESP-IDF prompt with IDF_PATH environment variable.
+        echo;
+        echo For example: setup_win.bat C:\SysGCC\esp32\esp-idf\v4.4
+        echo;
+        goto :ERR
+    )
+    if exist "%IDF_PATH%" (
+        echo Using IDF_PATH: %IDF_PATH%
+    ) 
+) else (
+    if not exist "%1" (
+        echo ERROR: optional directory was specified, but not found: %1
+        goto :ERR
+    )
+
+    SET IDF_PATH=%1
+    echo Using specified IDF_PATH: %IDF_PATH%
+)
+
+:: if no IDF_PATH is found, we don't know what to do. Go exit with error.
+if "%IDF_PATH%" == "" (
+  echo Please launch the script from ESP-IDF command prompt,
+  echo or set your desired IDF_PATH environment variable,
+  echo or pass a parameter to your directory, such as for VisualGDB with ESP-IDF 4.4:
+  echo;
+  echo   .\setup_win.bat C:\SysGCC\esp32\esp-idf\v4.4
+  echo;
+  goto :ERR
+)
+
+:: Here we go!
+:: setup some path variables
+echo;
+
+set SCRIPTDIR=%CD%
+set BASEDIR=%SCRIPTDIR%\..\..\..
+
+:: SSH
+set WOLFSSH_ESPIDFDIR=%BASEDIR%\IDE\Espressif\ESP-IDF
+
+set WOLFSSHLIB_TRG_DIR=%IDF_PATH%\components\wolfssh

```

Comment:
Can you quote the path? My path was "\Users\John Safranek", and was getting treated as multiple parameters to xcopy. (I did link my home directory to "\Users\John", which worked.)

Reply:
added quotes to paths